Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Multistage docker build, requires docker 17.05
+
+# builder stage
+FROM nvidia/cuda:10.0-devel as builder
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
This line never works since curl is not installed yet. It's correctly done in line 17

Reply:
you are right. Idk why I left that there when there was another line below with the same command.

Thanks